pkg/expr/sql_command.go

```diff
@@ -162,3 +181,57 @@ func totalCells(frames []*data.Frame) (total int64) {
 	}
 	return
 }
+
+func extractNumberSetFromSQLForAlerting(frame *data.Frame) ([]mathexp.Number, error) {
```

Comment:
A bit rough right now:
 - Should valid input more
 - Check for duplicates

Otherwise I think users will get off track with what is valid for alerting, or we will hit more errors upstream.

Think will defer this to future PR.